README.rst

```diff
@@ -40,6 +40,21 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb where you specify path for data and metastore. You need
+to ensure that the directories specified for data, wal and metastore exist and
+are set correctly.
```

Comment:
agreed. Also "Single-node influxdb where you specify paths for the data and metastore directories".